package.json

```diff
@@ -41,10 +38,12 @@
     "@angular-devkit/build-angular": "~0.1001.1",
     "@angular-devkit/build-ng-packagr": "~0.1001.1",
     "@angular/cli": "~10.1.1",
+    "@angular/compiler": "~10.1.1",
```

Comment:
I'm not certain if this should be in the devDependencies.

By default (unless I'm mistaken) the angular cli scaffolds this into the dependencies array.

Reply:
Ops, yes. Reverted.